app/containers/elements/Student/index.js

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import injectSaga from 'utils/injectSaga';
+import { createStructuredSelector } from 'reselect';
+import PropTypes from 'prop-types';
+
+import Button from 'components/elements/Button';
+import TextFieldComponent from '../../../components/elements/TextField/index';
+import { FieldsWrapper } from '../../../components/elements/Button/styles';
```

Comment:
I suggest that you create a styled div of your own with the same styles that Button has, but also adding `margin-left: 40%;` so that the form is centered(I've tried it and it aligns well) . Please do not add this change in the styles.js of Button because it may have impact on other pages. Thanks!